app/helpers/fhir_helper.rb

```diff
@@ -108,16 +117,28 @@ def patient_as_fhir(patient)
         to_datetime_extension(patient.expected_purge_ts, 'expected-purge-date'),
         to_string_extension(patient.exposure_risk_assessment, 'exposure-risk-assessment'),
         to_string_extension(patient.public_health_action, 'public-health-action'),
-        to_bool_extension(patient.contact_of_known_case, 'contact-of-known-case'),
-        to_string_extension(patient.contact_of_known_case_id, 'contact-of-known-case-id'),
-        to_string_extension(patient.member_of_a_common_exposure_cohort_type, 'common-exposure-cohort-name'),
         to_string_extension(patient.potential_exposure_location, 'potential-exposure-location'),
         to_string_extension(patient.potential_exposure_country, 'potential-exposure-country'),
         to_interpreter_required_extension(patient.interpretation_required),
         to_date_extension(patient.extended_isolation, 'extended-isolation'),
         to_string_extension(patient.monitoring_reason, 'reason-for-closure'),
         to_string_extension(patient.follow_up_reason, 'follow-up-reason'),
-        to_string_extension(patient.follow_up_note, 'follow-up-note')
+        to_string_extension(patient.follow_up_note, 'follow-up-note'),
+        to_string_extension(patient.additional_planned_travel_destination, 'additional-planned-travel-destination'),
+        to_string_extension(patient.additional_planned_travel_destination_state, 'additional-planned-travel-destination-state'),
+        to_string_extension(patient.additional_planned_travel_destination_country, 'additional-planned-travel-destination-country'),
+        to_string_extension(patient.additional_planned_travel_port_of_departure, 'additional-planned-travel-port-of-departure'),
+        to_string_extension(patient.additional_planned_travel_type, 'additional-planned-travel-type'),
+        to_string_extension(patient.case_status, 'case-status'),
+        to_datetime_extension(patient.closed_at, 'closed-at'),
+        to_string_extension(patient.gender_identity, 'gender-identity'),
```

Comment:
I don't think we can do that in this case, since I'm seeing the `Patient.gender` element as [required](https://www.hl7.org/fhir/patient-definitions.html#Patient.gender), so the value has to come from this [ValueSet](https://www.hl7.org/fhir/valueset-administrative-gender.html), which does not have sufficient values. But that blurb you linked did mention the existing `genderIdentity` [extension](https://www.hl7.org/fhir/extension-patient-genderidentity.html), which I think would make sense for us to use, since that can be bound to the Gender Identity Value Set, which as you said maps well to our fields.

Reply:
Ohh yeah good call, I didn't notice that that needed to be done with an extension, but yeah let's do that.

Reply:
Extension used in https://github.com/SaraAlert/SaraAlert/commit/3881a715e296d7607ca4ae30ad7a13805b9c569c.